components/apimgt/org.wso2.carbon.apimgt.rest.api.util/src/main/java/org/wso2/carbon/apimgt/rest/api/util/utils/OAuthTokenInfo.java

```diff
@@ -0,0 +1,96 @@
+/*
+ *
+ *  Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package org.wso2.carbon.apimgt.rest.api.util.utils;
+
+/**
+ * Details about an oauth Access Token.
+ */
+public class OAuthTokenInfo {
```

Comment:
Shouldn't we add this to `org.wso2.carbon.apimgt.api` module?

Reply:
+1 to move to api module for better usability